onnxruntime/core/providers/cpu/math/clip.cc

```diff
@@ -31,6 +32,17 @@ ONNX_CPU_OPERATOR_VERSIONED_KERNEL(
 
 REG_KERNEL_NONTEMPL(Clip, 12, Clip, float, double, int8_t, uint8_t, int64_t, uint64_t);
 
+template<typename T>
```

Comment:
If `Eigen` is put in `clip.h`, it will be included in CUDA kernel's header and trigger a NVCC bug.